centreon/src/Centreon/Infrastructure/Platform/PlatformRepositoryRDB.php

```diff
@@ -82,10 +82,11 @@ public function getWidgetsVersion(string $webVersion): array
             }
         }
 
-        $dashboardWidgetsRequest = $this->translateDbName('SELECT `name`, `version` FROM `:db`.dashboard_widgets');
+        $dashboardWidgetsRequest = $this->translateDbName('SELECT `name` FROM `:db`.dashboard_widgets');
+        $version = $this->getWebVersion();
         if (($statement = $this->db->query($dashboardWidgetsRequest)) !== false) {
             while ($result = $statement->fetch(\PDO::FETCH_ASSOC)) {
-                $versions[(string) $result['name']] = (string) $result['version'];
+                $versions[(string) $result['name']] = $version;
```

Comment:
According `getWebVersion` `$version` can be null, won't that be a problem ?

Reply:
Nope, that was already handle like this for other widgets or web version

Reply:
Nice spot :+1:

Reply:
@jeremyjaouen For the future we should rework `getWebVersion()' and throw an exception if no version is available